pkg/operator/workloadcontroller/workload_controller_openshiftapiserver_v311_00.go

```diff
@@ -397,3 +409,17 @@ func resourceSelectorForCLI(obj runtime.Object) string {
 	}
 	return kind + group + "/" + name
 }
+
+func proxyMapToEnvVars(proxyConfig map[string]string) []corev1.EnvVar {
+	envVars := []corev1.EnvVar{}
```

Comment:
move down where it is used, and return nil in the if-statement. Easier to read...